trunk/src/app/srs_app_statistic.cpp

```diff
@@ -495,12 +495,16 @@ srs_error_t SrsStatistic::dumps_vhosts(SrsJsonArray* arr)
     return err;
 }
 
-srs_error_t SrsStatistic::dumps_streams(SrsJsonArray* arr)
+srs_error_t SrsStatistic::dumps_streams(SrsJsonArray* arr, int start, int count)
 {
     srs_error_t err = srs_success;
-    
-    std::map<std::string, SrsStatisticStream*>::iterator it;
-    for (it = streams.begin(); it != streams.end(); it++) {
+
+    std::map<std::string, SrsStatisticStream*>::iterator it = streams.begin();
+    for (int i = 0; i < start + count && it != streams.end(); i++) {
```

Comment:
Iterator should be plus 1

Reply:
nice catch, I will create another PR to fix

Reply:
Oh sorry, my careless.

Reply:
My fault, I will create a PR ASAP

Reply:
@akanchi @duiniuluantanqin I have created a new PR in https://github.com/ossrs/srs/pull/2582, please help to review and sorry for the inconvenience